### PR TITLE
Adding IP addresses from all interfaces to initial checkin message

### PR DESCRIPTION
**Pull request by ahhh**
Adding IP addresses from all interfaces to initial checkin message

### CODE REVIEW

File: cmd/merlinagent/main.go [hunk 1]
```diff
@@ -124,6 +125,24 @@ func initialCheckIn(host string, client *http.Client) bool {
...
+			addrs, err := iface.Addrs()
+			if err == nil {
+				for _, addr := range addrs {
+					ips = append(ips, addr.String())
+				}
+			}
+		}
+	} else {
+		if debug {
+			color.Red("[!]There was an error getting the the ip addresses")
```
Comment (Ne0nd0g) [inline cmd/merlinagent/main.go]: Can we change `ip` to `IP`?

File: cmd/merlinserver/main.go [hunk 2]
```diff
@@ -41,7 +41,7 @@ import (
 	"github.com/satori/go.uuid"
 
 	// Merlin
-	"github.com/Ne0nd0g/merlin/pkg"
+	merlin "github.com/Ne0nd0g/merlin/pkg"
```
Comment (Ne0nd0g) [inline cmd/merlinserver/main.go]: Please remove the prepended `merlin` string. I believe it isn't required.

File: cmd/merlinserver/main.go [hunk 3]
```diff
@@ -321,6 +322,7 @@ func agentInitialCheckIn(j messages.Base, p messages.SysInfo) {
 	agents[j.ID].agentLog.WriteString(fmt.Sprintf("[%s]UserName: %s\r\n", time.Now(), p.UserName))
 	agents[j.ID].agentLog.WriteString(fmt.Sprintf("[%s]UserGUID: %s\r\n", time.Now(), p.UserGUID))
 	agents[j.ID].agentLog.WriteString(fmt.Sprintf("[%s]Process ID: %d\r\n", time.Now(), p.Pid))
+	agents[j.ID].agentLog.WriteString(fmt.Sprintf("[%s]Ips: %v\r\n", time.Now(), p.Ips))
```
Comment (Ne0nd0g) [inline cmd/merlinserver/main.go]: Can we change `Ips` to `IPs`?

File: cmd/merlinserver/main.go [hunk 4]
```diff
@@ -605,6 +607,7 @@ func shell() {
 							{"UserName", agents[a].userName},
 							{"User GUID", agents[a].userGUID},
 							{"Hostname", agents[a].hostName},
+							{"Ips", fmt.Sprintf("%v", agents[a].ips)},
```
Comment (Ne0nd0g) [inline cmd/merlinserver/main.go]: Can we change `Ips` to `IPs`?

**Comment by ahhh** [comment]
@Ne0nd0g changes made as requested!

**Comment by Ne0nd0g** [comment]
Thank you!

**Pull request merged**